service/awsconfig/v8/resource/cloudformation/desired.go

```diff
@@ -14,9 +14,16 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	mainStack, err := newMainStack(customObject)
-	if err != nil {
-		return nil, microerror.Mask(err)
+	var mainStack StackState
+	{
+		r.logger.LogCtx(ctx, "level", "debug", "message", "computing desired state for the guest cluster cloud formation stack")
+
+		mainStack, err = newMainStack(customObject)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		r.logger.LogCtx(ctx, "level", "debug", "message", "computed desired state for the guest cluster cloud formation stack")
```

Comment:
When developing and following logs I find this useful and informative.

Reply:
I'm fine with having them but I think there was also a suggestion to remove them?

As long as we're consistent across operators I'm fine with this. I'm also inclinded to have more rather than less logging.